packages/devtools_extensions/lib/src/template/extension_manager.dart

```diff
@@ -50,7 +56,9 @@ class ExtensionManager {
       _handleMessageListener = _handleMessage.toJS,
     );
 
-    // TODO(kenz): handle the ide theme that may be part of the query params.
+    setGlobal(IdeTheme, getIdeTheme());
+    theme.value = ideTheme;
```

Comment:
It is not clear to me how this solves the issue since we are only setting the notifier a single time and the listener in `_DevToolsExtensionState.initState` is actually getting added after this logic has finished.